index.css

```diff
@@ -381,3 +399,85 @@ form textarea {
   font-size: 15px;
   color: #172b4d;
 }
+
+@media (min-width: 768px) {
+  nav .menu-bar {
+    display: none;
+  }
+
+  nav {
+    justify-content: space-around;
+    gap: 240px;
+  }
+
+  nav .drop-down {
+    display: inline-block;
+    position: inherit;
+  }
+
+  .drop-down ul {
+    display: flex;
+    justify-content: space-between;
+    gap: 40px;
+  }
+
+  .drop-down ul li a {
+    color: #344563;
+  }
+
+  .headline-content {
+    width: 65%;
+  }
+
+  .headline-content h1 span::after {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .project-item:nth-of-type(1),
+  .project-item:nth-of-type(3) {
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .project-item:nth-of-type(2),
+  .project-item:nth-of-type(4) {
+    flex-direction: row-reverse;
+    gap: 20px;
+  }
```

Comment:
- Nice work you have done so far 👍 , but there are still some small differences between the Figma template and your design 😄. the width of your project cards doesn't exactly match the design. Please try to fix this. Know that replicating the exact specifications gives confidence to recruiters in your ability to design the exact webpage they'll require 🙏

Your project looks like this:

   - Please pay attention to the width of each of your project cards 🙏. The width in the Figma design takes a little bit more space than you have implemented in your design. I suggest you look into fixing this 👍 
 ![work card figma](https://user-images.githubusercontent.com/73607512/192861749-7b3cec64-fd42-4314-9cea-6c56de4db92a.PNG)

Figma template-
![work card student](https://user-images.githubusercontent.com/73607512/192861468-76365269-b5bd-4e78-baef-6c838f26660f.PNG)